app/assets/javascripts/ui_models/editor.ts

```diff
@@ -97,8 +96,9 @@ export class Editor {
   /**
    * Sets the editor contents by setting its note.
    */
-  public setNote(note: SNNote) {
+  public setNote(note: SNNote, isTemplate?: boolean) {
```

Comment:
Do you think this is better than just `isTemplate = false` in the signature and then instead of `this.isTemplateNote = isTemplate ?? false;` it would just be `this.isTemplateNote = isTemplate;`?

Reply:
True, forgot about it.